rules/check_field_names_use_lower_snake_case_test.go

```diff
@@ -45,9 +46,13 @@ func TestFieldNamesUseLowerSnakeCaseRule(t *testing.T) {
 	rule := checkFieldNamesUseLowerSnakeCase()
 
 	for _, test := range tests {
-		req := testutil.MustCreateRequestFromTemplate(tmpl, test)
-
 		errPrefix := fmt.Sprintf("Check field name `%s`", test.FieldName)
+
+		req, err := lint.NewProtoRequest(testutil.MustCreateFileDescriptorProtoFromTemplate("", tmpl, test, nil), nil)
```

Comment:
I have no clue figuring out what the arguments are in `MustCreateFileDescriptorProtoFromTemplate`. Maybe we need to encompass those arguments to a struct.

Reply:
IntelliJ automatically fills in the names of the parameters, so it's easy to see "filename", "srcTmpl", "data", "deps". I'm not against the idea of using a struct, but that also makes it a bit more cumbersome to use. Maybe I can restructure the functions to be more obvious.

Reply:
Decided to go with struct.